unstructured/metrics/table/table_eval.py

```diff
@@ -166,10 +194,8 @@ def process_file(self) -> TableEvaluation:
         )
 
         return TableEvaluation(
-            total_tables=total_tables,
-            table_level_acc=(
-                round(total_predicted_tables / total_tables, 2) if total_tables else np.nan
-            ),
+            total_tables=len(ground_truth_table_data),
+            table_level_acc=round(predicted_table_acc, 2),
```

Comment:
I was thinking about rounding at this level - as we're getting some aggregations on tens of tables in next step - is it ok to round it to 2 decimal places? We're losing some data then and AFAIK numbers shouldn't be rounded before aggregations (rounds can be added just before visualization).